package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@waku/rln",
-  "version": "0.0.14",
+  "version": "0.1.0",
```

Comment:
Big bump :)
Probably time to sort out automated release management?

Reply:
I think until we expect someone to use RLN in prod we leave it out. We can then reconsider when RLN is used beyond POCs